hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMultiMapProxy.java

```diff
@@ -148,6 +149,12 @@ public boolean remove(Object key, Object value) {
         MultiMapRemoveCodec.ResponseParameters resultParameters = MultiMapRemoveCodec.decodeResponse(response);
         return new UnmodifiableLazyList<V>(resultParameters.response, getSerializationService());
     }
+    public void delete(Object key) {
+        checkNotNull(key, NULL_KEY_IS_NOT_ALLOWED);
+
+        Data keyData = toData(key);
+        MultiMapDeleteCodec.encodeRequest(name, keyData, 4L);
```

Comment:
Magic number `4L`. Should this be the thread ID?

Reply:
Thanks, fixed.